app/vmalert/config/testdata/rules/vlog-rules0-good.rules

```diff
@@ -0,0 +1,29 @@
+groups:
+  - name: RequestCount
+    type: vlogs
```

Comment:
do we really want to introduce `vlogs` notion?

Reply:
We already have it for `graphite`. What is your take on this?

Reply:
yeap but we never introduced `vlogs` in our product victorialogs - yes, VL - yes

Reply:
@tenmozes We also add the flag `-rule.defaultRuleType` to allow user to set it for all groups. 
The `type` param is there already, and can be used to set up vmalert for multiple databases with different types through some proxies if user want.